lib/vm/src/trap/traphandlers.rs

```diff
@@ -459,6 +459,35 @@ pub unsafe fn wasmer_call_trampoline(
     })
 }
 
+/// Call the wasm function pointed to by `callee`, *not* wrapped into `catch_traps`.
+///
+/// This is a performance optimization for making function calls in batch.
+/// It must be called within a callback passed to `catch_traps` invocation.
+///
+/// * `vmctx` - the callee vmctx argument
+/// * `caller_vmctx` - the caller vmctx argument
+/// * `trampoline` - the jit-generated trampoline whose ABI takes 4 values, the
+///   callee vmctx, the caller vmctx, the `callee` argument below, and then the
+///   `values_vec` argument.
+/// * `callee` - the third argument to the `trampoline` function
+/// * `values_vec` - points to a buffer which holds the incoming arguments, and to
+///   which the outgoing return values will be written.
+///
+/// # Safety
+///
+/// Wildly unsafe because it calls raw function pointers and reads/writes raw
+/// function pointers.
```

Comment:
`# Safety` section should be updated with information about how to call this correctly (that is, how do you do the "check" that's being omitted)